README.md

```diff
@@ -48,13 +48,23 @@ The data in the specification contained in this document is intended for consump
 
 ## Overview of the Change Process
 GBFS is an open specification, developed and maintained by the community of producers and consumers of GBFS data.
-The specification is not fixed or unchangeable. As the bikeshare industry evolves, it is expected that the specification will be extended by the GBFS community to include new features and capabilities over time. To manage the change process, the following guidelines have been established.
-
-The general outline for changing the spec has 4 steps:
-1.	Propose a change by opening an issue at the GBFS GitHub repository.
-2.	Receive comments and feedback from the GBFS community and iterate on the proposed change. Discussion lasts for as long as the proposer  feels necessary, but must be at least 7 calendar days
-3.	Find at least one GBFS producer to implement and test the proposed change.
-4.	Submit a final request-for-comments on the proposed change to the issue discussion. If no outstanding issues are identified after one week’s time, and there is general agreement that the proposed change is worthwhile and follows the GBFS guiding principles outlined below, the proposal will be officially adopted.
+The specification is not fixed or unchangeable. As the shared mobility industry evolves, it is expected that the specification will be extended by the GBFS community to include new features and capabilities over time. To manage the change process, the following guidelines have been established.
+
+
+* A change is proposed by opening a Pull Request at the GBFS GitHub repository.A change is proposed by opening a Pull Request at the GBFS GitHub repository. The proposer becomes “The Advocate”. Comments and feedback from the GBFS community are received to iterate on the proposed change. Discussion lasts for as long as necessary to address questions and revisions, but must be at least 7 calendar days.
```

Comment:
"A change is proposed by opening a Pull Request at the GBFS GitHub repository" is repeated here

Reply:
Fixed!